app/views/v2/common_all_open_cases_view.scala.html

```diff
@@ -23,9 +23,9 @@
 
 @this()
 
-@(title: String, cases : CasesTabViewModel)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
+@(cases : CasesTabViewModel)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
```

Comment:
Could this just be called "open_cases_view"? Or is it common to several pages?

Reply:
👍